### PR TITLE
Developer portal

### PR DESCRIPTION
**Pull request by rob-kiers**
Added developer endpoint page
• Added developer hotlink to default page
• Added developer folder and index.md
• Added developer page specific CSS
• Added icons to assets\images

### CODE REVIEW

File: developer/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+introtekst over de verschillende endpoints die Kadaster Datasience team onderhoud. In de basis bieden wij linked dataen graphQL endpoints aan.
+
+<br/>
+
+<img class="developerpageIcon" src="/assets/images/linked-data_icon.png"> 
+## Linked Data
+Door linked data kunt u als developer datasets met elkaar verbinden. Bijvoorbeeld de dataset van de Basisregistratie Adressen en Gebouwen (BAG) met andere datasets. Omdat veel informatie via een adres gelinkt kan worden, leent de BAG zich hier goed voor.
+
+Voor meer informatie, kijk bij de <a href="https://labs.kadaster.nl/cases/architectuur-integraal-bevragen">Use Case: Aanpak en Architectuur van de (Kadaster) Knowledge Graph</a>
```
Comment (kad-vriesf) [inline developer/index.md:18]: Mooier om geen absolute link te maken maar /cases/archi...

File: developer/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+
+<img class="developerpageIcon" src="/assets/images/graphql_icon.svg">
+## GraphQL
+Met GraphQL ondervangen we een aantal vaak gestelde tekortkomingen van de services die we op onze basisregistraties leveren. De levering van data is vraaggestuurd in plaats van aanbodgestuurd, en daardoor kan de data beter op de gebruiker worden aangeboden. Verschillende datasets kunnen met een administratieve connectie integraal bevraagd worden.
+
+Voor meer informatie, kijk bij de <a href="https://labs.kadaster.nl/cases/graphql-endpoint">Use Case: GraphQL Open endpoint</a>
```
Comment (kad-vriesf) [inline developer/index.md:25]: Idem hier, /cases/graphql-

**Review by kad-vriesf — APPROVED** [review]
Akkoord na edit links

File: developer/index.md (hunk 1 above)
Comment (rob-kiers) [reply developer/index.md]: Updated link

File: developer/index.md (hunk 2 above)
Comment (rob-kiers) [reply developer/index.md]: Updated link

**Pull request merged**